ClientApp/src/components/pages/EmployeeDirectoryPage/EmployeeDirectoryPage.tsx

```diff
@@ -1,17 +1,91 @@
-import { PermissionsWrapper } from "components/reuseables";
-import React from "react";
-import { useRequireLogin } from "services";
-import { PermissionType } from "_generated";
+import {
+  AsyncComponent,
+  ErrorModal,
+  Header,
+  PermissionsWrapper,
+  useErrorHandling,
+} from "components";
+import _ from "lodash";
```

Comment:
Perhaps this identifier could be more descriptive?
```suggestion
import lodash from "lodash";
```

Reply:
haha the _ is a "low dash". But yeah, I can use better imports that don't import everything.